pkgs/build-support/trivial-builders.nix

```diff
@@ -509,8 +509,8 @@ rec {
       ''
         mkdir -p $out
         for i in $(cat $pathsPath); do
-          ${lndir}/bin/lndir -silent $i $out
-        done
+          ${lndir}/bin/lndir $i $out
+        done 2>&1 | sed 's/^/symlinkJoin: warning: keeping existing file: /'
```

Comment:
This is just a clever way to say "prepend every nonempty file"? Wow!

Reply:
more dirty than clever ...

considering the limitations of `lndir` (cannot replace/delete existing files), we should probably use a different tool for `symlinkJoin`